h/views/api/annotations.py

```diff
@@ -78,7 +79,9 @@ def create(request):
     schema = CreateAnnotationSchema(request)
     appstruct = schema.validate(_json_payload(request))
 
-    annotation = storage.create_annotation(request, appstruct)
+    annotation = request.find_service(AnnotationWriteService).create_annotation(
+        data=appstruct
```

Comment:
I've seen this name before in the H code base and not sure what it means.

I wound't change it now thought. It's probably "idiomatic H".